README.md

```diff
@@ -116,25 +116,31 @@ Expected output:
 trufflehog s3 --bucket=<bucket name> --only-verified
 ```
 
-## 5: Scan a Github Repo using SSH authentication in docker
+## 5: Scan S3 buckets using IAM Roles
+
+```bash
+trufflehog s3 --role-arn=<iam role arn>
```

Comment:
Should this have the `--bucket` flag?

Reply:
This specific example will enumerate all buckets a role has the authorization to list via the API.